db/migrate/20181130061333_add_personal_times.rb

```diff
@@ -0,0 +1,17 @@
+class AddPersonalTimes < ActiveRecord::Migration[5.1]
+  def change
+    create_table :course_personal_times do |t|
+      t.references :course_user, null: false, index: true, foreign_key: { to_table: :course_users }
+      t.references :lesson_plan_item,
+                   null: false, index: true, foreign_key: { to_table: :course_lesson_plan_items }
+      t.datetime :start_at, null: false
+      t.datetime :bonus_end_at
+      t.datetime :end_at
+      t.datetime :submitted_at
```

Comment:
`submitted_at` is on the `Course::Assessment::Submission` model.

Reply:
This was on the initial ERD proposal. I'm putting the submitted_at datetime here because otherwise it'll be a nightmare for the computation algorithms to keep joining to the assessment. We can migrate that column over?

Reply:
Yeah, all course users will still create a PersonalTime on submission even if they don't use "personalized timelines" to support the feature where we freeze the deadline on submission